theano/sandbox/cuda/tests/test_basic_ops.py

```diff
@@ -1093,6 +1093,9 @@ def test_adv_sub1_fast(self):
             self.assertTrue(val.ndim == data.ndim)
             utt.assert_allclose(val, good)
 
+    def test_noncontiguous_idx(self):
+        raise SkipTest("test doesn't work here")
```

Comment:
THe problem was that the test always use float64.

But this is the old back-end, so ok.